apps/backend/src/modules/user-routes/user-route.repository.ts

```diff
@@ -47,6 +47,39 @@ class UserRouteRepository implements Repository {
 		return UserRouteEntity.initialize(userRoute);
 	}
 
+	public async deleteSavedRoute(id: number, userId: number): Promise<boolean> {
```

Comment:
seems like usual delete to me. We only consider not_started routes as saved. Because those that were started will be shown in history and will automatically dissapear from saved 
```suggestion
	public async delete(id: number, userId: number): Promise<boolean> {
```

Reply:
@liza-veis the case is signature. delete expects one id argument